README.md

```diff
@@ -848,6 +848,9 @@ to change Zappa's behavior. Use these at your own risk!
         "cloudwatch_log_level": "OFF", // Enables/configures a level of logging for the given staging. Available options: "OFF", "INFO", "ERROR", default "OFF". C
         "cloudwatch_data_trace": false, // Logs all data about received events. Default false.
         "cloudwatch_metrics_enabled": false, // Additional metrics for the API Gateway. Default false.
+        "access_logging": false, // Add in logging of who accesses the API
```

Comment:
Can you keep the configuration options in alphabetical order please?

Reply:
Sure, just made the change.